misc/install-deps.sh

```diff
@@ -21,23 +21,28 @@ case $distro in
     "ubuntu" | "debian" | "raspbian" | "kali" | "linuxmint")
         apt-get $OPT install pandoc libdw-dev python3-dev libncursesw5-dev pkg-config
         apt-get $OPT install libluajit-5.1-dev || true
+        [ $DEBUG ] && apt-get $OPT install libunwind-dev || true
```

Comment:
We use `DEBUG=1` only during `make` stage as of now. I think it's better to provide a way to install libunwind as follows.
```
$ ./misc/install-deps.sh libunwind
```

Reply:
Nice! :)
I'm going to isolate the issue.